onnx/backend/test/cmd_tools.py

```diff
@@ -80,6 +86,8 @@ def parse_args():  # type: () -> argparse.Namespace
     subparser = subparsers.add_parser('generate-data', help='convert testcases to test data')
     subparser.add_argument('-o', '--output', default=DATA_DIR,
                            help='output directory (default: %(default)s)')
+    subparser.add_argument('-n', '--node', default=None,
+                           help='specify the node to exclude other testcases (default: %(default)s)')
```

Comment:
nit: we want the user to enter operator name which is referred to as op_type in rest of the code and more importantly here (https://github.com/onnx/onnx/blob/master/onnx/onnx-operators.proto#L101) 
so change from node to op_type to match the naming convention used everywhere else...

Reply:
specify the node to exclude other testcases -> generates test data only for the specified op_type

Reply:
Agree with the suggestion to rename the option to indicate it specifies an op name (not a node). Anything like "op_type" or "op" or "op_name" would be better than "node".

Reply:
I changed to use` --op_type`, but I don't know what the appropriate argument abbreviation is since `-o` is occupied... I chose '-t' eventually. Please let me know if you have better idea. Thanks.